packages/manager/modules/billing/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@ovh-ux/manager-billing",
+  "version": "0.0.0-alpha",
```

Comment:
I think we could use a `.prerelease` file for that
https://github.com/ovh/manager/blob/develop/scripts/common/repository.js#L243

Because I think the release script won't release an alpha.

Reply:
thanks, just added it here :
https://github.com/ovh/manager/pull/5121/files#diff-98c4c0a9eda678e34b5c6f1aa0521862a24274272e0a2d3c6b90274c492ac7c1R1